CODE_REVIEW.md

```diff
@@ -0,0 +1,14 @@
+## Code Review:
```

Comment:
Add description about the application, under code review heading and separate code smells into separate heading.
Try to point out more issue on data-access.
Try to remove unused functions or if used somewhere, try to minimise the code. Use pipe wherever possible.
Add reasons for code smells in code review and also implement the solution for all the code smells.